### ISSUE TITLE
Drop hints drawing over floating panes.

### ISSUE THREAD

**Issue by mjmacleod** [nhold/wxWidgets#51]
Hints should draw underneath floating panes not on top of them (I think?) - Don't think this should be fixed pre-merge just making a note of it for now - not sure how complex to fix...

**Comment by ghost** [comment]
Yes there is definetly something wrong with hints and we should leave this aside for now as it's not that easy. We need to think again about hinting, as I found it (and others here) not being clear at all. We have to find out a more accurate feedback to show where drop will happen, and under which form (normal dock or tab). Do you have any idea ?

**Comment by mjmacleod** [comment]
Think we are talking about two different issues here.
If you see the below image, what I am talking about is that the floating window has the hint drawn "on top of it" i.e. the floating window is also part blue, because it is floating it should probably float 'over' the hint. i.e. remain white.

![image](https://cloud.githubusercontent.com/assets/7540840/2942182/01dbcd4e-d9b1-11e3-99a6-4448a07f8d6d.png)

Will reply to your other comment on the other bug which I think deals more with what you are talking about.

**Comment by ghost** [comment]
I see what you mean now. No the hint should be over, because when you drag a large floating pane, it may obscure the drop area so much that it's hard to see where the drop occurs. So I think it's not an issue here, but a feature. Anyway this demonstrate the difficulty of having a good hinting.

**Comment by mjmacleod** [comment]
I agree there are various edge cases, so the best behaviour isn't clear (or there may actually be multiple desirable behaviours).
The whole hint part of the code needs to be more modular and 'changeable' I think - but all of this is probably best worried about later. So let us leave this alone for now, mostly just making a note of this for future reference.
